README.md

```diff
@@ -27,8 +27,56 @@ Enhanced Failure Monitoring (EFM) is a feature available from the [Host Monitori
 Please visit [this page](./docs/using-the-jdbc-driver/UsingTheJdbcDriver.md#using-the-aws-jdbc-driver-with-plain-rds-databases) for more information.
 
 ## Getting Started
-For more information on how to obtain the AWS JDBC Driver, minimum requirements to use it, and how to integrate the AWS JDBC Driver into your project, please visit the [Getting Started page](./docs/GettingStarted.md).
+For more information on how to obtain the AWS JDBC Driver, minimum requirements to use it, 
+and how to integrate the AWS JDBC Driver into your project, please visit the 
+[Getting Started page](./docs/GettingStarted.md).
+### Maven Central
+You can search on The Central Repository with GroupId and ArtifactId [software.amazon:aws-advanced-jdbc-wrapper][mvn-search].
 
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/software.amazon.jdbc/aws-advanced-jdbc-wrapper/badge.svg)](https://maven-badges.herokuapp.com/maven-central/software.amazon.jdbc/aws-advanced-jdbc-wrapper)
+```xml
+<!-- Add the following dependency to your pom.xml, -->
+<!-- replacing LATEST with specific version as required -->
+
+<dependency>
+  <groupId>software.amazon</groupId>
+  <artifactId>aws-advanced-jdbc-wrapper</artifactId>
+  <version>LATEST</version>
+</dependency>
+```
+
+[mvn-search]: https://search.maven.org/artifact/software.amazon/postgresql "Search on Maven Central"
+
+## Properties
+
```

Comment:
We should definitely add a note to ensure that backs the changes in the examples about setting properties values directly with objects instead of using strings

Are we sure that other pieces of documentation do not refer to properties with strings?